plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/FieldEmbedding.kt

```diff
@@ -53,6 +57,15 @@ class UserFieldEmbedding(
 ) : FieldEmbedding {
     override val accessPolicy: AccessPolicy = if (symbol.isVal) AccessPolicy.ALWAYS_READABLE else AccessPolicy.ALWAYS_INHALE_EXHALE
     override val includeInShortDump: Boolean = true
+
+    private val _typesContainingInPrimaryConstructor = mutableMapOf<TypeEmbedding, FirPropertySymbol>()
```

Comment:
Is it possible to only have these in the derived class?  I don't like it that `addType...` above doesn't actually always add the type.